kas/meta-coral.yml

```diff
@@ -21,6 +21,7 @@ repos:
     refspec: a4de91e7debb49b47e502b70527a6a91fb8cb9fc
     layers:
       meta:
+      meta-isar:
```

Comment:
Only Ubuntu and rasbios come from there.

Reply:
... and the `DISTRO_ARCH` variable, which is provided by the `qemuamd64.conf` (machine), which is also part of `meta-isar`.

Reply:
Then please fix the commit description, stating that you want to use `qemuamd64.conf` from meta-isar, nothing distro config related.

Reply:
done.